src/PointwiseFunctions/AnalyticData/GrMhd/SpecInitialData.cpp

```diff
@@ -280,10 +282,27 @@ template <size_t ThermodynamicDim>
 template <typename DataType>
 void SpecInitialData<ThermodynamicDim>::VariablesComputer<DataType>::operator()(
     const gsl::not_null<Scalar<DataType>*> electron_fraction,
-    const gsl::not_null<Cache*> /*cache*/,
+    const gsl::not_null<Cache*> cache,
     hydro::Tags::ElectronFraction<DataType> /*meta*/) const {
-  std::fill(electron_fraction->begin(), electron_fraction->end(),
-            constant_electron_fraction);
+  const auto& temperature =
+      cache->get_var(*this, hydro::Tags::Temperature<DataType>{});
+  const auto& rest_mass_density =
+      get<hydro::Tags::RestMassDensity<DataType>>(interpolated_data);
```

Comment:
I wonder if all these calls should also instead grab tho the cache? That way if the function that computes rho does some atmosphere flooring in the future everywhere else has that applied too.